### DISCUSSION TITLE
Alpha 143

### DISCUSSION THREAD

**Discussion by Eugeny** [Eugeny/tabby#4076] [General]
* Added hotkeys to loop through panes
* Added support for IPv6 addresses in SSH quick connect field
* Bracketed paste is now only used when support is advertised by the shell
* SFTP/Zmodem: added size and speed display for transfers
* SSH: fixed agent authentication
* SSH: added an option to reuse private keys stored in the Vault
* SSH: added an option to force the SSH agent connection type on Windows (Pageant vs OpenSSH)
* Fixed `Ctrl-C` not getting broadcast into all panes
* Prevented "history restored" from popping up in new SSH panes

<hr /><em>This discussion was created from the release <a href='https://github.com/Eugeny/terminus/releases/tag/v1.0.143'>Alpha 143</a>.</em>

**Comment by SavuBogdan** [comment]
The current implementation of reusable vault keys only solves the file transfer problem , as is right now it's not very useful because now I have all my 50 connections tied down to a private key I cannot update , so if I change key I will still have to manually go in each connection and change key ... , imo the desirable solution would be a single point of change.
